README.md

```diff
@@ -26,339 +26,213 @@
-* [Repository Agents](docs/repository_agents.md)
-* [Test](docs/test.md)
-
-Triton can be [built using
-Docker](docs/build.md#building-triton-with-docker) or [built without
-Docker](docs/build.md#building-triton-without-docker). After building
-you should [test Triton](docs/test.md).
-
-It is also possible to [create a Docker image containing a customized
-Triton](docs/compose.md) that contains only a subset of the backends.
-
-The Triton project also provides [client libraries for Python and
-C++](https://github.com/triton-inference-server/client) that make it
-easy to communicate with the server. There are also a large number of
-[example clients](https://github.com/triton-inference-server/client)
-that demonstrate how to use the libraries. You can also develop your
-own clients that directly communicate with Triton using [HTTP/REST or
-GRPC protocols](docs/inference_protocols.md). There is also a [C
-API](docs/inference_protocols.md) that allows Triton to be linked
-directly into your application.
-
-A [Triton backend](https://github.com/triton-inference-server/backend)
-is the implementation that executes a model. A backend can interface
-with a deep learning framework, like PyTorch, TensorFlow, TensorRT or
-ONNX Runtime; or it can interface with a data processing framework
-like [DALI](https://github.com/triton-inference-server/dali_backend);
-or you can extend Triton by [writing your own
-backend](https://github.com/triton-inference-server/backend) in either
-[C/C++](https://github.com/triton-inference-server/backend/blob/main/README.md#triton-backend-api)
-or
-[Python](https://github.com/triton-inference-server/python_backend).
-Triton also allows developers to write backends that can send multiple
-responses for a request or not send any responses for a request. Backends
-and models that operate in this way are referred to as [decoupled
-backends and models](docs/decoupled_models.md).
-
-A [Triton repository agent](docs/repository_agents.md) extends Triton
-with new functionality that operates when a model is loaded or
-unloaded. You can introduce your own code to perform authentication,
-decryption, conversion, or similar operations when a model is loaded.
-
-## Additional Resources
-
-* Additional documentation, presentations, and examples are located in the
-  [NVIDIA Developer Zone](https://developer.nvidia.com/nvidia-triton-inference-server).
-
-* Specific end-to-end examples for popular models, such as ResNet, BERT, and
-  DLRM are located in the [NVIDIA Deep Learning Examples page on
-  GitHub](https://github.com/NVIDIA/DeepLearningExamples).
+configuration](docs/model_configuration.md) for the model.  
+
+- Optimize your models setting [scheduling and batching](docs/architecture.md#models-and-schedulers)
+  parameters and [model instances](docs/model_configuration.md#instance-groups).
+- Use the [Model Analyzer tool](https://github.com/triton-inference-server/model_analyzer)
```

Comment:
Are we trying to list these points in an order of the steps they should be performed?
If yes, the correct order in which these steps should be:
3, 4, 1, 2, 5?

Reply:
Wasn't thinking that, but it is a good point. re-ordered.